packages/pharos/src/components/table/pharos-table.scss

```diff
@@ -0,0 +1,45 @@
+:host {
+  display: inline-flex;
+  flex-direction: column;
+}
+
+.table {
+  border-collapse: collapse;
+
+  th {
+    border: 1px solid var(--pharos-color-marble-gray-base);
+    padding: var(--pharos-spacing-1-x);
+  }
+
+  td {
+    border: 1px solid var(--pharos-color-marble-gray-base);
+    padding: var(--pharos-spacing-1-x);
+  }
+}
+
+.table-controls {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-top: var(--pharos-spacing-one-half-x);
+
+  .item-per-page-wrapper {
+    display: flex;
+    align-items: center;
+    column-gap: var(--pharos-spacing-one-half-x);
+
+    .item-per-page-selector {
+      width: 75px;
+    }
+  }
+
+  .pagination {
+    height: var(--pharos-spacing-one-and-a-half-x);
+  }
+}
+
+.hide-table-caption {
+  visibility: hidden;
```

Comment:
This CSS style will actually hide the content from a screen reader, similar to `display: none`. It's better to use the following set of styles when trying to visually hide content: 

```
.visually-hidden:not(:focus):not(:active) {
  clip: rect(0 0 0 0); 
  clip-path: inset(50%);
  height: 1px;
  overflow: hidden;
  position: absolute;
  white-space: nowrap; 
  width: 1px;
}
```

Reply:
I think this mixin is what you want within pharos https://github.com/ithaka/pharos/blob/e396a023873a9c5c949343919aee174570a9acd4/packages/pharos/src/utils/scss/_mixins.scss#L130

Reply:
done!